docs/HOWTO.md

```diff
@@ -1,45 +1,34 @@
 
-Those issues are tracked more closely due to their potential wide impact on users, organizations or the Web architecture.
+* If *-needs-resolution is applied to an issue, automatic tooling will remove the *-tracker label.
 
-# Horizontal issues and W3C Process
+## Do these labels replace the need to ask for horizontal review?
 
-Horizontal issues should be raised and tracked as early possible. Some get raised before a specification is adopted by a W3C Working Group to be on the Standard track.
+No.  The is *-tracker label asks for input on a specific issue.  For through review, follow [the published process for requesting horitzontal review](https://www.w3.org/wiki/DocumentReview#How_to_get_horizontal_review)
 
-From a W3C Process perspective, the Director pays additional attention to horizontal issues before approving a new maturity level for a specification.
 
-Note: In the implementation of the Process 2020, the importance of an issue will more directly impact whether a Group is allowed to get an automatic approval from the Director.
+For horizontal review groups
+============================
 
-# How do W3C track horizontal issues?
+* WGs may apply the *-tracker label to issues in their own repositories to draw your attention to specific issues. 
 
-W3C tracks horizontal issues using a set of GitHub labels, allowing both specification contributors and horizontal reviewers to flag an issue as an horizontal issue. The level of importance of such issue may vary depending on the opinion of an horizontal group.
+* You may also apply the *-tracker label to issues you want to follow.
 
-# What's the difference between -tracker and -needs-resolution?
+* Apply the *-needs-resolution label to issues you expect to be resolved before a specification moves to a new maturity level.
 
-Each horizontal group can a designation to allow grouping of horizontal issues and easy tracking by horizontal group. In each group, there is currently 2 GitHub labels to reflect the degree of importance of such issue.
+* When either label is applied in a WG repository, a tracking issue will be created in your own repository within about a day.  Such new issues are also marked as 'pending'.  You may remove the 'pending' label after triaging the issue.
 
-* *-tracker indicates that this issue should be brought to the attention of the horizontal group.
-* *-needs-resolution is used by an horizontal group to indicate their expectation before a specification is allowed to more to a new maturity level.
+* Alternatively, you may apply the needs-resolution label to an existing tracking issue to upgrade it.  
 
-Note: while the distinction between the two labels is not relevant from a W3C Process perspective, it is intended to help horizontal groups make their intent clear when sorting issues.
+* Whenever *-needs-resolution or needs-resolution is applied to an existing tracked issue, an automatic tool will remove the -tracking and tracked labels (in a few hours).
```

Comment:
Does it? Do you need to change the WG issue or the tracker issue, or either?  Note that changing needs-resolution to tracker should NOT automatically change the other label (dangerous).